spinoso-time/src/time/tzrs/convert.rs

```diff
@@ -0,0 +1,77 @@
+use core::fmt::{Display, Formatter, Result};
+
+use super::{Time, ToA};
+
+impl Display for Time {
+    /// Returns a conanocial string representation of _time_
+    ///
+    /// Can be used to implement [`Time#asctime`], [`#Time#ctime`], [`Time#to_s`] and
+    /// [`Time#inspect`]
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use spinoso_time::tzrs::Time;
+    /// let now = Time::utc(2022, 05, 26, 13, 16, 22, 0);
+    /// assert_eq!(now.to_string(), "2022-05-26 13:16:22 UTC");
+    /// ```
+    ///
+    /// [`Time#asctime`]: https://ruby-doc.org/core-2.6.3/Time.html#method-i-asctime
+    /// [`Time#ctime`]: https://ruby-doc.org/core-2.6.3/Time.html#method-i-ctime
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        // TODO: future
+        //self.strftime("%Y-%m-%d %H:%M:%S %z")
```

Comment:
let's turn this TODO into a ticket.

Reply:
is this `strftime` format string the default `Time#to_s` output? If so that's amazing and I'd love to see it documented with some `irb` repl examples like:

https://github.com/artichoke/artichoke/blob/06e9e0d88dc7d7451dc5b2269e9d05c69bef5da6/artichoke-backend/src/extn/core/string/trampoline.rs#L125-L142

Reply:
> is this strftime format string the default Time#to_s output?
Yep! (Or at least in the case of ruby yep!)

- https://ruby-doc.org/core-2.6.3/Time.html#method-i-inspect
- https://ruby-doc.org/core-2.6.3/Time.html#method-i-to_s

I'll throw it in the PR for integrating this into `artichoke-backend`

Note: I saw something somewhere about the to_s being locale specific - but I'm not sure where I saw that :thinking: 

Side note, apart form chrono, there doesn't appear to be any pure rust crate for strftime. I might take some inspiration from tz-rs + tzdb (for localisation). Would the artichoke org want to take on the package?

Reply:
Created relevant issue: https://github.com/artichoke/artichoke/issues/1914